<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginSuccessfulClass</class>
 <widget class="QMainWindow" name="LoginSuccessfulClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Neflix</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label_username">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>371</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::WinPanel</enum>
    </property>
    <property name="text">
     <string>username</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_searchMovie">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Search by name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_searchMovieByCategory">
    <property name="geometry">
     <rect>
      <x>232</x>
      <y>40</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Search by category</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget_movies">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>70</y>
      <width>371</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_preferences">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>80</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Your preferences</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_wishlist">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>120</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Wishlist</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_recommandations">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>160</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Recommandations</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
